packages/typespec-autorest/test/decorators.test.ts

```diff
@@ -129,6 +129,7 @@ describe("typespec-autorest: decorators", () => {
             [AutorestTestLibrary.name]: {
               ...options,
               "emitter-output-dir": outputDir,
+              "output-file": "openapi.json",
```

Comment:
if we don't set it explicitly, we will get the default output-file `{azure-resource-provider-folder}/{service-name}/{version-status}/{version}/openapi.json`, which is not true in this test case.